lib/pdksync/utils.rb

```diff
@@ -343,6 +343,13 @@ def self.create_filespace
       configuration.pdksync_dir
     end
 
+    # @summary
+    #   This method when called will create a directory identified by the set global variable 'configuration.pdksync_gem_dir', on the condition that it does not already exist.
+    def self.create_filespace_gem
+      FileUtils.mkdir_p configuration.pdksync_dir unless Dir.exist?(configuration.pdksync_gem_dir)
```

Comment:
it is creating the pdksync_dir, not the pdksync_gem_dir

Reply:
@carabasdaniel Hi Dany ,all gems will be cloned to separate directory  pdksync_gem_dir

Reply:
oh got it. Changed the code.